### PR TITLE
BaSP-TG-40: timesheet POST and PUT tests

### PR DESCRIPTION
**Pull request by lucasrendo**

### CODE REVIEW

File: src/tests/time-sheets.test.js [hunk 1]
```diff
@@ -1 +1,393 @@
...
+        endDate: '2022-07-07T00:00:00.000+00:00',
+      },
+    );
+
+    expect(response.body.msg).toMatch('Code 400: No employee with the id');
+  });
+
+  // INVALID DATA
+
+  test('Ivalid task id', async () => {
```
Comment (ArianaMaldonado) [inline src/tests/time-sheets.test.js:101]: Invalid*

File: src/tests/time-sheets.test.js [hunk 2]
```diff
@@ -1 +1,393 @@
...
+        projectId: '6283b662e53ed4c648db8a45',
+        startDate: '2022-06-05T00:00:00.000+00:00',
+        endDate: '2022-07-07T00:00:00.000+00:00',
+      },
+    );
+
+    expect(response.status).toBe(400);
+  });
+
+  test('Ivalid employee id', async () => {
```
Comment (ArianaMaldonado) [inline src/tests/time-sheets.test.js:119]: Same here

File: src/tests/time-sheets.test.js [hunk 3]
```diff
@@ -1 +1,393 @@
...
+        projectId: '6283b662e53ed4c648db8a45',
+        startDate: '2022-06-05T00:00:00.000+00:00',
+        endDate: '2022-07-07T00:00:00.000+00:00',
+      },
+    );
+
+    expect(response.status).toBe(400);
+  });
+
+  test('Ivalid project id', async () => {
```
Comment (ArianaMaldonado) [inline src/tests/time-sheets.test.js:137]: And here

File: src/tests/time-sheets.test.js [hunk 4]
```diff
@@ -1 +1,393 @@
...
+        projectId: '6283b662e53ed4c648db8a45',
+        startDate: '2022-12-05T00:00:00.000+00:00',
+        endDate: '2021-07-07T00:00:00.000+00:00',
+      },
+    );
+
+    expect(response.status).toBe(400);
+  });
+
+  test('Ivalid end date', async () => {
```
Comment (ArianaMaldonado) [inline src/tests/time-sheets.test.js:209]: Idem

File: src/tests/time-sheets.test.js [hunk 5]
```diff
@@ -1 +1,393 @@
...
+          '62891835d4e286802a02756e',
+          '62891835d4e286802a02756f',
+          '62891835d4e286802a02756g',
+        ],
+        projectId: '6283b662e53ed4c648db8a45',
+        startDate: '2022-06-05T00:00:00.000+00:00',
+        endDate: '2022-07-07T00:00:00.000+00:00',
+      },
+    );
+    // expect(response.body.msg).toBe('Code 400: Timesheet already exists');
```
Comment (ArianaMaldonado) [inline src/tests/time-sheets.test.js:259]: Why is this commented?

File: src/tests/time-sheets.test.js [hunk 6]
```diff
@@ -1 +1,393 @@
...
+        employeeId: '62894106842279ad30cd964c',
+      },
+    );
+
+    expect(response.body.msg).toMatch('Code 400: No employee with the id');
+  });
+
+  // INVALID DATA
+
+  test('Ivalid task id', async () => {
```
Comment (ArianaMaldonado) [inline src/tests/time-sheets.test.js:335]: Invalid*

**Review by dayanagm96 — APPROVED** [review]
When you fix Ari's comments everything should be fine

**Review by RicardoDosko — APPROVED** [review]

**Review by ArianaMaldonado — APPROVED** [review]

**Pull request merged**